src/post_profile_point.ts

```diff
@@ -0,0 +1,51 @@
+import { DynamoDB, PutItemCommand } from '@aws-sdk/client-dynamodb';
+
+const dynamodb = new DynamoDB({});
+const TABLENAME = process.env.TABLENAME || "";
+
+// the profileId#conference is separated by symbol '#'.
+const separotorChar = '#'
+
+export const handler = async (event: any = {}): Promise<any> => {
+    
+    const profileId =  Number(event.profileId);
+    if (!profileId) {
+            throw new Error('Error400: cannot get profileId');
+    }
+    
+    const point =  Number(event.point);
+    if (!point) {
+            throw new Error('Error400: cannot get point');
+    }
+
+    const conference = String(event.conference);
+    if (!conference) {
+        throw new Error('Error400: cannot get conference')
+    }
+
+    const reasonId = Number(event.reasonId);
+    if (!reasonId) {
+        throw new Error('Error400: cannot get reason')
+    }
+
+    // Timezone is UTC.
+    const timestamp = Date.now();
+    
+    try {
+        const command = new PutItemCommand({
+            TableName: TABLENAME,
+            Item: {
+                'profileId#conference': { S: String(profileId)+ separotorChar + String(conference)},
```

Comment:
nits:

```suggestion
                'profileId#conference': { S: [profileId, conference].map(String).join(separatorChar)},
```

```suggestion
                'profileId#conference': { S: `${profileId}#${conference}` },
```

好みですが、こういう書き方もできます。
（後者は、separatorを一箇所でしか使わないなら、変数定義せずにそのまま書いても良いのでは、という提案も含みます）